### ISSUE TITLE
`Tools / Extract slice` doesn't work on surfaces

### ISSUE THREAD

**Issue by kephale** [haesleinhuepf/napari-segment-blobs-and-things-with-membranes#29]
Since the name `Extract slice` is generic I thought it might work for any napari layer. However, the widget does indicate that it only works on image layers though. 

I ran into this after running into https://github.com/haesleinhuepf/napari-process-points-and-surfaces/issues/84, so I started looking for a GUI way to slice the surface in time.

**Comment by haesleinhuepf** [comment]
Yeah, extracting a slice from a surface mesh is really a different task, also presumably located in a different Tools menu in Napari. Thus, I'd say this is out-of-scope for nbatwm. Thanks the feedback anyway!

**Comment by kephale** [comment]
I would just follow up with a suggestion that maybe the name should be clarified. Since the `Tools` menu contains functions for many layer types (with very precise names in most cases), `Utilities` doesn't imply layer type, but `Extract slice` is particularly vague given that it sounds like it will continue to just work for Images.
